monkey/monkey_island/cc/services/zero_trust/monkey_findings/monkey_zt_details_service.py

```diff
@@ -23,7 +23,9 @@ def fetch_details_for_display(finding_id: ObjectId) -> dict:
             details = details[0]
             details['latest_events'] = MonkeyZTDetailsService._get_events_without_overlap(details['event_count'],
                                                                                           details['latest_events'])
-        return details
+            return details
+        else:
+            return {}
```

Comment:
Are you sure this is the behavior you want?
I would AT LEAST add a warning log, if not raise an exception

Reply:
You're right, it looks like all monkey ZT findings have events.